apps/docs-snippets2/src/cookbook/custom-contract-calls.ts

```diff
@@ -0,0 +1,61 @@
+import { bn, buildFunctionResult, Contract, Provider, Wallet } from 'fuels';
+
+import { LOCAL_NETWORK_URL, WALLET_PVT_KEY } from '../env';
+import { CounterFactory } from '../typegend';
+
+const provider = await Provider.create(LOCAL_NETWORK_URL);
+const wallet = Wallet.fromPrivateKey(WALLET_PVT_KEY, provider);
+const deploy = await CounterFactory.deploy(wallet);
+const { contract } = await deploy.waitForResult();
+
+const receiverWallet = Wallet.generate({ provider });
+
+// #region custom-transactions-contract-calls
```

Comment:
Should this come before the imports?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3367/commits/aefcdf3b96c79de8f7f509928e00e3d2b3dd9f7c